verified-inference/host/api/repository/enclave_repository.py

```diff
@@ -0,0 +1,60 @@
+import asyncio
+from enum import Enum
+import json
+import subprocess
+from typing import Dict, Optional
+from utils import proxy_manager
+
+
+class EnclaveState(Enum):
+    RUNNING = "running"
+    STOPPED = "stopped"
+
+
+class Enclave:
+    proxy_task: Optional[asyncio.Task] = None
+
+    def __init__(self, enclave_name: str, state: EnclaveState):
+        self.enclave_name = enclave_name
+        self.state = state
+
+
+class EnclaveRepository:
+    _enclaves: Dict[str, Enclave] = {}
+
+    def __init__(self):
+        with subprocess.Popen(
+            ["/bin/nitro-cli", "describe-enclaves"], stdout=subprocess.PIPE
```

Comment:
Similar comment to [this one](https://github.com/galadriel-ai/Sentience/pull/36/files#r1908601875) - I don't think repository should know the such low-level details of connecting to TEE.

Reply:
Note: can be abstracted in another PR, shouldn't block merging this one.

Reply:
My understanding of repository is to handle the low-level and external stuff like DB etc. If not in repository, where else do you think is the best place to encapsulate these implementations?

Reply:
In my head, the protocol between host and tee is a library, not related to the specifics of our project. Imagine that in the future we would open-source (no prio now ofc)